pages/main-page.js

```diff
@@ -5,7 +5,7 @@ import { getAvailableLocale, tl, tlByte } from "../shared/translation";
 import PixxioJsdk from "./components/pixxio-jsdk";
 import UpdateResourcesComponent from "./components/update-resources";
 
-const apiKey = "iM91iRu6kb86Y6IaWMsK9T9q7";
+const apiKey = "1675955190applicationShop8sy2CfW366";
```

Comment:
Der muss auch rotiert werden und über ne Umgebungsvariable reinkommen:

```suggestion
const apiKey = process.env.PIXXIO_API_KEY;
```